Documentation/howto/revert-a-faulty-merge.txt

```diff
@@ -14,7 +14,7 @@ How to revert a faulty merge
 
 Alan <alan@clueserver.org> said:
 
-    I have a master branch.  We have a branch off of that that some
+    I have a master branch.  We have a branch off of it that some
```

Comment:
That's not a word duplication.

Reply:
(Both of these non-word-duplications should be factored out into their own commit, and the commit message should probably make a case for the change. For example, I am not all that certain that we should edit what Alan said. [He said what he said](https://www.youtube.com/watch?v=Cc6dQPFKh28)...)

Reply:
Indeed